data/pilots/first-order/tie-wi-whisper-modified-interceptor.json

```diff
@@ -0,0 +1,134 @@
+{
+  "name": "TIE/wi Whisper Modified Interceptor",
+  "xws": "tiewiwhispermodifiedinterceptor",
+  "size": "Small",
+  "ffg": 0,
+  "dial": [
+    "1TW",
+    "1YW",
+    "2TW",
+    "2BB",
+    "2FB",
+    "2NB",
+    "2YW",
+    "3LR",
+    "3TW",
+    "3BB",
+    "3FB",
+    "3NB",
+    "3YW",
+    "3PR",
+    "4FB",
+    "4KR",
+    "5FB",
+    "5KR"
+  ],
+  "dialCodes": ["???"],
+  "faction": "First Order",
+  "stats": [
+    { "arc": "Bullseye Arc", "type": "attack", "value": 3 },
+    { "arc": "Single Turret Arc", "type": "attack", "value": 2 },
+    { "type": "agility", "value": 2 },
+    { "type": "hull", "value": 3 },
+    { "type": "shields", "value": 2 }
+  ],
+  "actions": [
+    { 
+      "difficulty": "White", 
+      "linked": { "difficulty": "White", "type": "Rotate Arc" },
+      "type": "Focus" 
+    },
+    { "difficulty": "White", "type": "Evade" },
+    { "difficulty": "White", "type": "Lock" },
+    { 
+      "difficulty": "White",
+      "linked": { "difficulty": "White", "type": "Rotate Arc" },
+      "type": "Barrel Roll"
+    },
+    { 
+      "difficulty": "White",
+      "linked": { "difficulty": "White", "type": "Rotate Arc" },
+      "type": "Boost"
+    }
+  ],
+  "pilots": [
+    {
+      "name": "Kylo Ren",
+      "xws": "kyloren-tiewiwhispermodifiedinterceptor",
+      "initiative": 5,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
```

Comment:
I'm not sure if we know the slots for all the pilots yet. Presumably Kylo has one talent and one force? Probably better to leave slots out entirely until release for the pilots we don't know